src/screens/OtherProfile/index.js

```diff
@@ -340,7 +261,7 @@ const OtherProfile = () => {
       );
     };
     const __renderFollowerDetail = () => {
-      if (blockStatus.blocker) return <></>;
+      if (isBlocking) return <></>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `blockStatus.blocker` has been replaced with `isBlocking`. Ensure that `isBlocking` provides the same functionality as `blockStatus.blocker`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->